platform/windows/os_windows.cpp

```diff
@@ -1332,7 +1332,11 @@ void OS_Windows::vprint(const char* p_format, va_list p_list, bool p_stderr) {
 	if (p_stderr)
 		fwprintf(stderr,L"%s",wbuf);
```

Comment:
I think that could have a bug too, it's just my execution path did not reach that for some condition.